paasc/src/lib.rs

```diff
@@ -12,24 +14,27 @@ static CIPHERSUITES: &[&SupportedCipherSuite; 5] = &[
     &ciphersuite::TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
 ];
 
-fn buf_read(path: &str) -> Result<BufReader<File>, Box<dyn std::error::Error>> {
+fn buf_read(path: impl AsRef<Path>) -> Result<BufReader<File>, Error> {
```

Comment:
minor nit: se the `anyhow::Result` alias here and in other places for good practice